float64 xCenter
float64 yCenter
float64 zCenter
float64 width
float64 length
float64 height
string Class
geometry_msgs/Point[] points
